www/assets/app.css

```diff
@@ -22,8 +22,47 @@ h1, h4 {
 h1 {
   border-bottom: 2px solid #222;
   text-align: left;
+  line-height: 1.5;
 }
 
+button {
+  border-radius: 5px;
+  background: #CCC;
+  border: none;
+}
+
+p button {
+  font-size: 0.8em;
+  font-weight: bold;
+  padding: 3px 5px;
+}
+
+input,
+textarea,
+table{
+  border: 1px solid #aaa;
+}
+
+textarea{
+  height: 3.25em;
+  max-width: 100%;
+}
+
+button:hover,
+button:focus,
+input[type="text"]:hover,
+input[type="text"]:focus,
+input[type="number"]:focus,
+input[type="number"]:hover,
+textarea:hover,
+textarea:focus{
+  border-color: transparent;
+  outline: none;
+  box-shadow: 0 0 0 3px #222;
```

Comment:
Standard outline:
<img width="584" alt="bildschirmfoto 2016-01-02 um 12 37 26" src="https://cloud.githubusercontent.com/assets/391124/12074341/c16b1302-b14d-11e5-92ef-d4068abe6e99.png">
Current version:
<img width="583" alt="bildschirmfoto 2016-01-02 um 12 37 41" src="https://cloud.githubusercontent.com/assets/391124/12074343/ca1ea7ca-b14d-11e5-82c5-792ecb2d5681.png">

I'd say the ugly version is actually less obvious, tbh. But I'll submit to the majority :)